### PR TITLE
Move DS config and allow to run jest in example folder

### PR DESCRIPTION
**Pull request by mfrachet**
# Update build for DS and jest for the cra-example

## Description

- Allows to import DS components independently
- Allows to run jest tests in the cra-examples folder

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -38,6 +39,9 @@
     ]
   },
   "devDependencies": {
-    "jest-styled-components": "^7.0.4"
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.7",
+    "jest-styled-components": "^7.0.4",
+    "prettier": "^2.3.0"
```
Comment (mfrachet) [inline example/package.json:45]: ℹ️ prettier is necessary to run toMatchInlineSnapshot

File: example/src/ContentManager/EditView/Information.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import React from "react";
-import { Box, Stack, Text, Row, Divider } from "@strapi/design-system";
+import { Box } from "@strapi/design-system/Box";
+import { Stack } from "@strapi/design-system/Stack";
```
Comment (soupette) [inline example/src/ContentManager/EditView/Information.js:3]: What is the default export in the lib?

Comment (mfrachet) [reply example/src/ContentManager/EditView/Information.js:3]: there's no default exports

**Review by HichamELBSI — APPROVED** [review]
LGTM good job :)

**Pull request merged**